feconf.py

```diff
@@ -362,6 +362,13 @@ def get_empty_ratings():
 # the existing storage models for UserStatsModel.
 DASHBOARD_STATS_DATETIME_STRING_FORMAT = '%Y-%m-%d'
 
+# If the default dashboard is set to the value below, it signifies the
+# preferred dashboard of the user is the creator dashboard.
+DASHBOARD_TYPE_CREATOR = 'creator'
```

Comment:
These should go in the shared constants.js file since they're used in both frontend and backend.